File: Python/tests/test_cgt_cm.py
import pytest

from hbayesdm.models import cgt_cm


def test_cgt_cm():
    _ = cgt_cm(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_cgt_cm.R
context("Test cgt_cm")
library(hBayesDM)

test_that("Test cgt_cm", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(cgt_cm(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
